### PR TITLE
show book request detail

### PR DESCRIPTION
**Pull request by banguyenvareal**

### CODE REVIEW

File: app/controllers/book_requests_controller.rb [hunk 1]
```diff
@@ -20,6 +20,10 @@ def index
       BookRequest.order(created_at: :desc).page(params[:page]).per(16)
   end
 
+  def show
+    @book_request = BookRequest.first
```
Comment (dinhnguyencanh) [inline app/controllers/book_requests_controller.rb]: why is `BookRequest.first`

File: app/views/book_requests/_images.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<% if images.empty? %>
+  <img id="main_image" src='<%= image_path('default-book-cover.jpg') %>'>
+<% else %>
+  <img id="main_image" src='<%= images.first.url %>'>
```
Comment (dinhnguyencanh) [inline app/views/book_requests/_images.html.erb]: fefine a method.
`def get_thumb_url(book_request)`
`  # return thumb`
`end`

File: app/assets/stylesheets/book_request.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+.detail {
+  padding-left: 30px;
+  margin-top: 15px;
+}
+
+hr {
+  border-style: inset;
+  border-width: 1px;
+  padding-left: 0px;
```
Comment (dinhnguyencanh) [inline app/assets/stylesheets/book_request.scss]: remove

File: app/assets/stylesheets/book_request.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  margin-left: -15px;
+}
+
+#main_image {
+  width: 380px;
+  height:400px;
+}
+
+.small-image{
+  padding-left: 0px;
```
Comment (dinhnguyencanh) [inline app/assets/stylesheets/book_request.scss]: remove

**Review by dinhvareal — APPROVED** [review]

**Review by hieunguyentrung — APPROVED** [review]

**Pull request merged**